src/pages/app.astro

```diff
@@ -1,16 +1,12 @@
 ---
 import BaseLayout from "@layouts/BaseLayout.astro";
-import App from "@components/page/App.svelte";
+import AppLanding from "@components/page/App/AppLanding.svelte";
 ---
 
 <style lang="scss">
   @use "../styles/" as *;
 </style>
 
-<BaseLayout
-  title="Get the Future Super app"
-  description="Future Super has a mobile app for Apple and Android devices"
-  image="https://res.cloudinary.com/future-super/image/upload/v1666674820/app-preview.png"
```

Comment:
I think you need to keep these otherwise the 'preview' won't show up properly in social, google, slack etc